cleanup_pods/command_line.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import logging
+from kubernetes import client, config
+
+# https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/
+POD_PHASE = [
+    # All Containers in the Pod have terminated in success, and will not be restarted.
+    'succeeded',
+    # All Containers in the Pod have terminated, and at least one Container has terminated in failure.
+    # That is, the Container either exited with non-zero status or was terminated by the system.
+    'failed'
```

Comment:
@ashb @schnie what about `restartPolicy`? 
If restartPolicy is:
`Always`: Restart Container; Pod phase stays `Running`.
`OnFailure`: Restart Container; Pod phase stays `Running`.
`Never`: Pod phase becomes `Failed`.
should i delete pod in every situation? or only when `restartPolicy==Never`?

Reply:
Looking at the `RestartPolicy` makes sense to me. I think we just want to make sure we leave core airflow pods alone if they happen to be crashing.